src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { type FC, type MouseEvent, type ChangeEvent } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getExtendedSearchParams } from '../../helpers/getExtendedSearchParams';
+import { getSearchParamsWithout } from '../../helpers/getSearchParamsWithout';
+import Select from '../common/Select/Select';
+import NavItem from './NavItem/NavItem';
+import Paginate from '../common/Paginate/Paginate';
+import type { NavBarProps } from '../../types/types';

```

Comment:
you can use @alias
https://medium.com/slackernoon/use-typescript-aliases-to-clean-up-your-import-statements-7210b7ec2af1